<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Color_Manipulation</class>
 <widget class="QDialog" name="Color_Manipulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="Contenedor" native="true">
     <widget class="QWidget" name="colorWheel_1" native="true">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>241</width>
        <height>241</height>
       </rect>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="HUE1"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>180</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="SAT2"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="HUE2"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>210</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="VAL2"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_13">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="SAT1"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_14">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="VAL1"/>
      </widget>
     </widget>
     <widget class="QWidget" name="colorWheel_2" native="true">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>241</width>
        <height>241</height>
       </rect>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>180</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="SAT4"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="SAT3"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>210</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="VAL4"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="HUE3"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_11">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="HUE4"/>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_12">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="VAL3"/>
      </widget>
     </widget>
     <widget class="QComboBox" name="operationBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Substract</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Change</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>310</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>SOURCE COLOR</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>310</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TO CHANGE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>H</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>S</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>H</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>S</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>130</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>H</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>S</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>280</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>V</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>220</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>H</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>250</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>S</string>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>350</y>
        <width>156</width>
        <height>23</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
